packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -66,6 +68,7 @@ describe("TinyliciousClient", () => {
         const containerAndServicesP = tinyliciousClient.getContainer("containerConfig", schema);
 
         const errorFn = (error) => {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
Ok this is cool.  This covers a major pet peeve of mine which is that the catch variable is of type `any`.  I was motivated to try to upgrade to tsc 4.5 where catch variables are `unknown` by default to address this very problem.  But I never got around to it, and this lint rule is even more encompassing.  Nice.

Reply:
Mentioning TypeScript upgrade issue for posterity:  #8572